docs/cloud/execution/overview.md

```diff
@@ -159,7 +159,7 @@ LocalAgent(labels=["dev", "staging"]).start()
 # Flow will be picked up by this agent
 ```
 
-On the other hand if you deploy a flow that has environment labels set to `["dev", "staging"]` and run an Agent with the labels `["dev"]` then it will not pick up the flow because there exists labels in the environment which were not provided to the agent.
+On the other hand, a Flow with environment labels `["dev", "staging"]` would not be run by an Agent with the label `["dev"]`. This is because the Flow is has a label not also provided to the Agent.
```

Comment:
```suggestion
On the other hand, a Flow with environment labels `["dev", "staging"]` would not be run by an Agent with the label `["dev"]`. This is because the Flow has a label not also provided to the Agent.
```